desc/configuration.py

```diff
@@ -1012,11 +1066,46 @@ def iota(self, new):
     @property
     def i_l(self):
         """Coefficients of iota profile (ndarray)."""
-        return self.iota.params
+        try:
+            return self.iota.params
+        except AttributeError:
+            return None
 
     @i_l.setter
     def i_l(self, i_l):
-        self.iota.params = i_l
+        try:
+            self.iota.params = i_l
+        except AttributeError:
```

Comment:
I think in this case we should throw an error saying you're trying to set something that doesn't exist